Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -27,6 +28,21 @@ protected void Update()
         {
             //TODO: Not sure if this works with 2 cameras?? Wait until later (using x negative scaling for now)
             interactTransform.LookAt(Camera.main.transform);
```

Comment:
TODO: connect with the CameraManager i made to get the current camera transform

Reply:
Was able to connect it to the CameraManager!